dbms/src/Storages/DeltaMerge/Index/RoughCheck.h

```diff
@@ -40,81 +40,93 @@ using Cmp = ValueComparision<Op>;
 
 
 template <typename T>
```

Comment:
Can we move the `template <typename T>` to the function `check` instead of the struct?

Reply:
Good idea! Fixed~